src/zepben/evolve/model/cim/iec61970/base/wires/create_wires_components.py

```diff
@@ -0,0 +1,484 @@
+    PowerElectronicsUnit: power_electronics_connection, max_p, min_p
+    BatteryUnit: battery_state, rated_e, stored_e
+    """
+    bu = BatteryUnit(**locals())
+    add_power_electronics_connection_connection(bu, usage_points, equipment_containers, operational_restrictions, current_feeders,
+                                                power_electronics_connection)
+    return bu
+
+
+# noinspection PyShadowingBuiltins,PyShadowingNames
+def create_breaker(mrid: str = None, name: str = '', description: str = "", names: List[Name] = None, location: Location = None,
+                   asset_info: AssetInfo = None, in_service: bool = True, normally_in_service: bool = True, usage_points: List[UsagePoint] = None,
+                   equipment_containers: List[EquipmentContainer] = None, operational_restrictions: List[OperationalRestriction] = None,
+                   current_feeders: List[Feeder] = None, base_voltage: BaseVoltage = None, terminals: List[Terminal] = None, _open: int = 0,
+                   _normally_open: int = 0) -> Breaker:
+    """
+    Breaker(ProtectedSwitch(Switch(ConductingEquipment(Equipment(PowerSystemResource(IdentifiedObject))))))
+    IdentifiedObject: mrid, name, description, names
+    PowerSystemResource: location, asset_info
+    Equipment: in_service, normally_in_service, usage_points, equipment_containers, operational_restrictions, current_feeders
+    ConductingEquipment: base_voltage, terminals
+    Switch: _open, _normally_open
+    ProtectedSwitch:
+    Breaker:
+    """
+    b = Breaker(**locals())
+    add_conducting_equipment_connection(b, usage_points, equipment_containers, operational_restrictions, current_feeders, terminals)
+    return b
+
+
+def create_busbar_section(mrid: str = None, name: str = '', description: str = "", names: List[Name] = None, location: Location = None,
+                          asset_info: AssetInfo = None, in_service: bool = True, normally_in_service: bool = True, usage_points: List[UsagePoint] = None,
+                          equipment_containers: List[EquipmentContainer] = None, operational_restrictions: List[OperationalRestriction] = None,
+                          current_feeders: List[Feeder] = None, base_voltage: BaseVoltage = None, terminals: List[Terminal] = None) -> BusbarSection:
+    """
+    BusbarSection(Connector(ConductingEquipment(Equipment(PowerSystemResource(IdentifiedObject)))))
+    IdentifiedObject: mrid, name, description, names
+    PowerSystemResource: location, asset_info
+    Equipment: in_service, normally_in_service, usage_points, equipment_containers, operational_restrictions, current_feeders
+    ConductingEquipment: base_voltage, terminals
+    Connector:
+    BusbarSection:
+    """
+    bs = BusbarSection(**locals())
+    add_conducting_equipment_connection(bs, usage_points, equipment_containers, operational_restrictions, current_feeders, terminals)
```

Comment:
the connection functions are not named consistently. `add_conducting_equipment_connecton` vs `add_common_equiment_connections`. Prefer dropping `_common`.